packages/notifi-node/lib/client/NotifiClient.ts

```diff
@@ -46,21 +46,21 @@ class NotifiClient {
     jwt,
     { key, walletPublicKey, walletBlockchain, ...payload },
   ) => {
-    const message = newSimpleHealthThresholdMessage(payload);
-    const input = {
-      walletPublicKey,
-      walletBlockchain,
-      messageKey: key,
-      messageType: message.type,
-      message: JSON.stringify(message.payload),
-    };
+      const message = newSimpleHealthThresholdMessage(payload);
+      const input = {
```

Comment:
I might disable my linter so I don't disrupt the formatting

Reply:
We might just want to use the `prettier` extension to adopt the prettierrc config’s convention (may not need to disable `eslint` as long as there is not dup config)
--> Install the vscode `prettier plugin` and  add the line below into vscode `settings.json`
"prettier.configPath": "./.prettierrc",
The way, the projects’ format can be adopted